drydock_backups/patches/k8s-jobs

```diff
@@ -152,3 +152,18 @@ spec:
                       requests:
                         storage: {{ BACKUP_K8S_EPHEMERAL_VOLUME_SIZE }}
           {%- endif %}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mongodb-backup-job
+  labels:
+    app.kubernetes.io/component: job
+spec:
+  ttlSecondsAfterFinished: 100
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: mongodb
+        image: {{ DOCKER_IMAGE_MONGODB }}
```

Comment:
What are we using this job for?

Reply:
Is the template to load [this job](https://github.com/eduNEXT/drydock-backups/pull/6/files#diff-c29b52c098c0d4017c4220701b2fead863755038c94927810dadfd58b1d5dba5R103). drydock is adding one mongodb job template but is to remove the dependency with drydock